include/seqan3/range/container/concept.hpp

```diff
@@ -204,20 +158,18 @@ SEQAN3_CONCEPT SequenceContainer = requires (type val, type val2, type const cva
     // modify container
     // TODO: how do you model this?
     // { val.emplace(typename type::const_iterator{}, ?                                   } -> typename type::iterator;
+    { val.insert(val.cbegin(), val2.front())                                           } -> typename type::iterator;
+    { val.insert(val.cbegin(), typename type::value_type{})                            } -> typename type::iterator;
+    { val.insert(val.cbegin(), typename type::size_type{}, typename type::value_type{})} -> typename type::iterator;
+    { val.insert(val.cbegin(), val2.begin(), val2.end())                               } -> typename type::iterator;
+    requires detail::is_basic_string_v<type> || requires(type val)
```

Comment:
this is still required?

Reply:
an improvement in any case

Reply:
> This isn't going to be backported, because of the new symbols. So fixed for GCC 9 only.

https://gcc.gnu.org/bugzilla/show_bug.cgi?id=83328#c8